### DISCUSSION TITLE
Merry Christmas Dave

### DISCUSSION THREAD

**Discussion by wsirons** [PlummersSoftwareLLC/NightDriverStrip#183] [General]
I just wanted to thank you for helping to make Christmas, or any celebration that can use some awesome lighting, much more festive with your Nightdriver code.  I've had a wonderful time setting up lighting strips and fairy light around the house, and modifying the code to suit the location.

Thank you so much for efforts, support, and magnificent attitude.  You have been an inspiration to me, and I'm sure, many others who are budding developers or possibly learning for themselves that all of our life's development exists on a spectrum.

Sorry if this is not the correct place to reach out like this, but just had to wish you, and the rest of this fantastic community, the merriest of Christmas's and a well lit New Year!

WS

**Comment by davepl** [comment]
Thanks for the kind words!  I’m just glad people are starting to find and use it!

Cheers,
Dave
